src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,186 @@
+  margin-bottom: $iui-xs;
+  margin-right: $iui-xs;
+
+  @supports (gap: $iui-s) {
+    margin-bottom: 0;
+    margin-right: 0;
+  }
+
+  @include themed {
+    box-shadow: inset 0 0 0 $iui-xxs
+      rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+    border: 1px solid t(iui-color-background-1);
+  }
+
+  &.iui-active {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs t(iui-color-foreground-primary);
+    }
+  }
+
+  &:hover {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs
+        rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5)),
+        inset 0 0 0 $iui-xxs
+        rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+    }
+  }
+
+  &.iui-add-color-button {
+    @include iui-focus;
+    @include iui-button-borderless;
+    box-shadow: none;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+
+    svg {
+      width: $iui-icons-default;
+      height: $iui-icons-default;
+      @include themed {
+        fill: t(iui-icons-color-actionable);
+      }
+    }
+
+    &:hover {
+      @include themed {
+        background-color: t(iui-color-background-1-overlay);
+        border-color: t(iui-color-background-1-overlay);
+      }
+
+      svg {
+        @include themed {
+          fill: t(iui-icons-color-actionable-hover);
+        }
+      }
+    }
+  }
+}
+
+@mixin iui-color-palette {
+  //Contains the color swatches
+  display: flex;
+  flex-wrap: wrap;
+  @supports(gap: $iui-s) {
+    gap: $iui-s;
+  }
+}
+
+@mixin iui-color-selection-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: $iui-baseline;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+}
+
+@mixin iui-color-field {
+  position: relative;
+  cursor: crosshair;
+  width: $iui-color-field-size;
+  height: $iui-color-field-size;
+
+  @include themed {
+    background-image: linear-gradient(0deg, rgb(0, 0, 0), transparent),
+      linear-gradient(90deg, rgb(255, 255, 255), var(--color));
+  }
+}
+
+@mixin iui-color-slider {
+  margin-left: $iui-sm;
+  position: relative;
+  width: $iui-s;
+  border-radius: $iui-border-radius;
+
+  @include themed {
+    background: linear-gradient(
+      $iui-color-system-red 0%,
+      $iui-color-system-yellow 17%,
+      rgb(0, 255, 0) 33%,
+      rgb(0, 255, 255) 50%,
+      $iui-color-system-blue 67%,
+      $iui-color-system-magenta 83%,
+      $iui-color-system-red 100%
+    );
+  }
+}
+
+@mixin iui-color-dot {
+  position: absolute;
+  top: var(--top, 0%);
+  left: var(--left, 0%);
+  width: $iui-m;
+  height: $iui-m;
+  border-radius: 50%;
+  transform: translate(-8px, -8px);
+  @include themed {
+    box-shadow: rgba(t(iui-color-foreground-accessory-rgb), t(iui-opacity-1)) 0
+      0 0 $iui-xxs - 1,
+      inset 0 0 0 $iui-xxs - 1
+      rgba(t(iui-color-foreground-accessory-rgb), t(iui-opacity-6));
+    background-color: var(--selected-color);
+  }
+
+  &.iui-white {
+    transform: translate(-4px, -9px);
+    cursor: grab;
```

Comment:
You might want to add also grabbing when it is `:active`. Check slider example (https://itwin.github.io/iTwinUI/backstop/minified/slider.html)

Reply:
I added grabbing to :active